File: C/beginner/1041.c
/*https://judge.beecrowd.com/pt/problems/view/1041*/
/*Including stdio*/
#include <stdio.h>

/*Main function*/
int main() {
    /*Initializing variables*/
    float x, y;

    /*Scanning*/
    scanf("%f %f", &x, &y);

    /*Finding quadrant*/
    if (x > 0 && y > 0) {
        printf("Q1\n");
    } else if (x < 0 && y > 0) {
        printf("Q2\n");
    } else if (x < 0 && y < 0) {
        printf("Q3\n");
    } else if (x > 0 && y < 0) {
        printf("Q4\n");
    } else if (x == 0 && y == 0) {
        printf("Origem\n");
    } else if (x == 0) {
        printf("Eixo Y\n");
    } else if (y == 0) {
        printf("Eixo X\n");
    } else {
        printf("Erro\n");
    }

    /*Ending*/
    return 0;
}

File: Haskell/beginner/1041.hs
-- https://judge.beecrowd.com/pt/problems/view/1041
-- Getting printf
import Text.Printf

-- Main Function
main = do
    -- Requesting values and assign into variables
    usr_input <- getLine
    let points = map read (words usr_input) :: [Double]
    let [a, b] = points

    -- Finding Quadrant
    if a > 0 && b > 0
        then printf "Q1\n"
    else if a < 0 && b > 0
        then printf "Q2\n"
    else if a < 0 && b < 0
        then printf "Q3\n"
    else if a > 0 && b < 0
        then printf "Q4\n"
    else if a == 0 && b == 0
        then printf "Origem\n"
    else if a == 0
        then printf "Eixo Y\n"
    else if b == 0
        then printf "Eixo X\n"
    else
        printf "Erro\n"

-- By Fernando Serra
-- https://github.com/fernandosserra